dot/network/notifications.go

```diff
@@ -256,13 +291,38 @@ func (s *Service) broadcastExcluding(info *notificationsProtocol, excluding peer
 				}
 			}
 
+			hsData, has = info.getHandshakeData(peer)
+			if !has {
+				continue
+			}
+
 			// we've already completed the handshake with the peer, send message directly
 			logger.Trace("sending message", "protocol", info.protocolID, "peer", peer, "message", msg)
-			err = s.host.send(peer, info.protocolID, msg)
-		}
 
-		if err != nil {
-			logger.Debug("failed to send message to peer", "peer", peer, "error", err)
+			err = s.host.writeToStream(hsData.stream, msg)
+			if err != nil {
+				logger.Trace("failed to send message to peer", "peer", peer, "error", err)
+			}
```

Comment:
this can be moved outside the `if-else` case, since in the `if` block we exchange the handshake, we still want to send the message after exchanging the handshake

Reply:
Done